### ISSUE TITLE
pip install from github is not working

### ISSUE THREAD

**Issue by emvalbuena** [mmngreco/sphinx-automated#11]
The installation pointing towards the Github repository,
`pip install git+git@github.com:mmngreco/sphinx-automated.git`, is not working at the moment. 

```bash
$ pip install git+git@github.com/mmngreco/sphinx-automated.git
Invalid requirement: 'git+git@github.com/mmngreco/sphinx-automated.git'
Traceback (most recent call last):
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\packaging\requirements.py", line 93, in __init__
    req = REQUIREMENT.parseString(requirement_string)
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\pyparsing.py", line 1654, in parseString
    raise exc
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\pyparsing.py", line 1644, in parseString
    loc, tokens = self._parse( instring, 0 )
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\pyparsing.py", line 1402, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\pyparsing.py", line 3417, in parseImpl
    loc, exprtokens = e._parse( instring, loc, doActions )
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\pyparsing.py", line 1406, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\pyparsing.py", line 3205, in parseImpl
    raise ParseException(instring, loc, self.errmsg, self)
pip._vendor.pyparsing.ParseException: Expected stringEnd (at char 3), (line:1, col:4)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_internal\req\constructors.py", line 253, in install_req_from_line
    req = Requirement(req)
  File "D:\Users\emillan\envs\ns\lib\site-packages\pip\_vendor\packaging\requirements.py", line 96, in __init__
    requirement_string[e.loc:e.loc + 8], e.msg
pip._vendor.packaging.requirements.InvalidRequirement: Parse error at "'+git@git'": Expected stringEnd

```

**Comment by asdf8601** [comment]
the problema still exists?

**Comment by asdf8601** [comment]
I have checked and the problem exists. This is due to a mistake in the README.md file.

The correct command should be:

```bash
pip install git+ssh://git@github.com/mmngreco/sphinx-automated.git
```

**Comment by asdf8601** [comment]
closed in 2387958
